.github/actions/build-rpm/src/main.ts

```diff
@@ -0,0 +1,140 @@
+import * as core from '@actions/core'
+import * as exec from '@actions/exec'
+import * as fs from 'fs'
+import * as path from 'path'
+import * as gpg from './gpg';
+import {
```

Comment:
Are we using all the functions from ./util? 
If yes, can we `import * as util from './util';`